roles/ceph-rgw/tasks/main.yml

```diff
@@ -48,3 +49,9 @@
     - containerized_deployment
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
+
+- name: include rgw_pool_pgs.yml
```

Comment:
why is `rgw_pool_pgs.yml` included twice?

Reply:
Your right, my mistake.